eZ/Publish/Core/Repository/NotificationService.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository;
+
+use eZ\Publish\API\Repository\NotificationService as NotificationServiceInterface;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use eZ\Publish\Core\Base\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\SPI\Persistence\Notification\Handler;
+
+class NotificationService implements NotificationServiceInterface
+{
+    /** @var Handler $persistenceHandler */
+    protected $persistenceHandler;
+
+    /** @var Repository $kernelRepository */
+    protected $kernelRepository;
+
+    /**
+     * NotificationService constructor.
+     *
+     * @param Handler $persistenceHandler
+     * @param Repository $kernelRepository
+     */
+    public function __construct(Handler $persistenceHandler, Repository $kernelRepository)
+    {
+        $this->persistenceHandler = $persistenceHandler;
+        $this->kernelRepository = $kernelRepository;
+    }
+
+    /**
+     * Get currently logged user notifications.
+     *
+     * @param int $limit Number of notifications to get
+     * @param int $page Notifications pagination
+     *
+     * @return \EzSystems\Notification\SPI\Persistence\ValueObject\Notification[]
+     */
+    public function getUserNotifications($limit, $page)
+    {
+        $currentUser = $this->kernelRepository->getCurrentUser();
+
+        $notifications = $this->persistenceHandler->getNotificationsByOwnerId($currentUser->id, $limit, $page);
+
+        return $notifications;
```

Comment:
`return $this->persistenceHandler->getNotificationsByOwnerId($currentUser->id, $limit, $page)`